Change obsolete values.h by float.h (for FLT_MAX)
Add sys/time.h for definition of timeval


--- codels/analyzeTarget.c.orig	2013-06-04 14:27:04.000000000 +0200
+++ codels/analyzeTarget.c	2013-06-05 17:37:45.000000000 +0200
@@ -26,7 +26,8 @@
 #include <stdlib.h>
 #include <string.h>
 #include <math.h>
-#include <values.h>
+#include <float.h>
+#include <sys/time.h>
 
 #include <portLib.h>
 #include "server/velodyneHeader.h"
